docs/documentation/development/bindings/faq.md

```diff
@@ -21,18 +21,18 @@ Here is a list of frequently asked question around the development of bindings.
     
     If your system does not provide you any possibility to get hold of such information, but rather only shows you a "logical" view of it, then 1) is also a valid option to pursue.
   
-  1. _Do I have to create XML files in `ESH-INF/thing` for all devices or is there any other option?_
+2. _Do I have to create XML files in `ESH-INF/thing` for all devices or is there any other option?_
```

Comment:
It may be valid, but currently the enumeration is not rendered properly (have a look at the live [Binding Development FAQ](https://eclipse.org/smarthome/documentation/development/bindings/faq.html) page). After I applied the changes to the enumeration and built the docs, numbers were generated correctly inside this article.